backend/clubs/views.py

```diff
@@ -2121,6 +2122,21 @@ def get_object(self):
         return user
 
 
+class UserUUIDAPIView(generics.RetrieveAPIView):
+    """
+    get: Retrieve the uuid for the given user.
+    """
+
+    queryset = get_user_model().objects.all()
+    serializer_class = UserUUIDSerializer
+    permission_classes = [IsAuthenticated]
+    http_method_names = ["get"]
+
+    def get_object(self):
+        user = self.request.user
+        return user
+
+
```

Comment:
An alternative to this could be making an action under `UserUpdateAPIView` instead of a brand new endpoint.

There are no problems with your code, but the proposed solution might be cleaner, since both endpoints share the same `queryset`, `permission_classes`, and `get_object` functionality.

Reply:
Oh ok I thought we wanted a separate endpoint just for this, but that makes sense.